### PR TITLE
anup: init at 0.4.0

### PR DESCRIPTION
**Pull request by natto1784**
This is my first package, sorry for any mistakes ;-;

###### Motivation for this change
I wanted an anime manager for anilist, found this, thought i should add this here too

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by natto1784** [comment]
@ofborg build anup

File: pkgs/applications/misc/anup/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  version = "0.4.0";
+  src = fetchFromGitHub {
+    owner = "Acizza";
+    repo = "anup";
+    rev = version;
+    sha256 = "sha256-4pXF4p4K8+YihVB9NdgT6bOidmQEgWXUbcbvgXJ0IDA=";
+  };
+
+  buildInputs = [
+    sqlite.dev
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/anup/default.nix:14]:
```suggestion
    sqlite
```

File: pkgs/applications/misc/anup/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ lib, rustPlatform, fetchFromGitHub, sqlite, xdg_utils }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "anup";
+  version = "0.4.0";
+  src = fetchFromGitHub {
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/anup/default.nix:6]:
```suggestion
  version = "0.4.0";

  src = fetchFromGitHub {
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/anup/default.nix (hunk 1 above)
Comment (natto1784) [reply pkgs/applications/misc/anup/default.nix]: done

File: pkgs/applications/misc/anup/default.nix (hunk 2 above)
Comment (natto1784) [reply pkgs/applications/misc/anup/default.nix]: added break

File: pkgs/applications/misc/anup/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{ lib, rustPlatform, fetchFromGitHub, sqlite, xdg_utils }:
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/anup/default.nix:1]:
```suggestion
{ lib, rustPlatform, fetchFromGitHub, sqlite, xdg-utils }:
```

File: pkgs/applications/misc/anup/default.nix [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  src = fetchFromGitHub {
+    owner = "Acizza";
+    repo = "anup";
+    rev = version;
+    sha256 = "sha256-4pXF4p4K8+YihVB9NdgT6bOidmQEgWXUbcbvgXJ0IDA=";
+  };
+
+  buildInputs = [
+    sqlite
+    xdg_utils
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/anup/default.nix:16]:
```suggestion
    xdg-utils
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Also please squash all package related commits.

File: maintainers/maintainer-list.nix [hunk 5]
```diff
@@ -7185,6 +7185,12 @@
     githubId = 818502;
     name = "Nathan Yong";
   };
+  natto1784 = {
```
[the remark this answers is not in the document]
Comment (natto1784) [reply maintainers/maintainer-list.nix:7188]: done

**Pull request merged**